bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -180,4 +181,126 @@
 		<label>Scene</label>
 		<description>The scene channel allows recalling a scene to all lights that belong to the scene.</description>
 	</channel-type>
+
+	<!-- CLIP 2 Channels -->
+	<channel-type id="button-last-event">
+		<item-type>Number</item-type>
+		<label>Button Last Event</label>
+		<description>Numeric code (e.g. 1003) representing the last push button event.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="motion-enabled">
+		<item-type>Switch</item-type>
+		<label>Motion Enabled</label>
+		<description>Motion detector enabled.</description>
+		<category>Lock</category>
+	</channel-type>
+
+	<channel-type id="light-level-2">
+		<item-type>Number:Illuminance</item-type>
+		<label>Light Level</label>
+		<description>Current light level.</description>
+		<state readOnly="true" pattern="%.0f %unit%"/>
+	</channel-type>
+
+	<channel-type id="light-level-enabled">
+		<item-type>Switch</item-type>
+		<label>Light Level Enabled</label>
+		<description>Light level sensor enabled.</description>
+		<category>Lock</category>
+	</channel-type>
+
+	<channel-type id="temperature-enabled">
+		<item-type>Switch</item-type>
+		<label>Temperature Sensor Enabled</label>
+		<description>Temperature sensor enabled.</description>
+		<category>Lock</category>
+	</channel-type>
+
+	<channel-type id="zigbee-status">
+		<item-type>String</item-type>
+		<label>Zigbee Status</label>
+		<description>Zigbee connection status.</description>
+		<category>QualityOfService</category>
```

Comment:
Umm. I disagree. The Zigbee status is absolutely an indication of "Quality of Service" .. albeit that it has three discrete text states rather than numeric bars. .. I think that perhaps a better solution would be to map the current three text states to numeric values so that Quality of Service dynamic icon will actually dynamically update. => WDYT?

Reply:
Given that (IIRC) you said only `CONNECTED` and `CONNECTIVITY_ISSUE` can happen in practice (which are also mapped to the thing status), I wonder whether this channel is useful at all?

Also: would `UNIDIRECTIONAL_INCOMING` by a larger or smaller value than `CONNECTIVITY_ISSUE`?

Reply:
> I think that perhaps a better solution would be to map the current three text states to numeric values so that Quality of Service dynamic icon will actually dynamically update. => WDYT?

Not a bad idea. You will still get the proper labels through the options. But how will you map your 4 values ? Meaning what icon for each value ? CONNECTED with all bars filled and DISCONNECTED with no bars filled. But what for CONNECTIVITY_ISSUE and UNIDIRECTIONAL_INCOMING ?

Reply:
> how will you map your 4 values 

Suggestion below..

- `connected` device has been recently been available => 4 bars
- `unidirectional incoming` the device only talks to bridge => 3 bars
- `connectivity issues` the device is powered off or has network issues = 1 bar (??) 
- `disconnected` => 0 bars

Reply:
Ok, why not. Take care the value should be between 0 and 4.
Is it ok for you @maniac103  ?

Reply:
Well, my opinion is right above your question ;-) I guess we both wrote at the same time.
As mentioned, for a given device the value of this channel always is a binary information (connected or not connected); the actual value just differs between devices (e.g. lights have 'connected', switches have 'unidirectional_incoming'). That binary information is already mapped into the thing status, so IMHO this channel doesn't add any value at all. If I'm missing something, please let me know.

Reply:
> switches have 'unidirectional_incoming'

My switches report 'connected'. I don't see unidirectional_incoming in real life. Perhaps it applies to (not yet existing) Zigbee low power switches??

> this channel doesn't add any value at all

I wonder if it makes sense for diagnostic purposes? Perhaps an 'advanced' channel? And we could use connected => 4 bars resp. !connected => 0 bars.

Reply:
> I wonder if it makes sense for diagnostic purposes?

IMHO it doesn't ... for diag purposes, writing a DEBUG log line every time the ZigBee status changes should be sufficient, I think?

> And we could use connected => 4 bars resp. !connected => 0 bars.

What value does this provide that the thing status doesn't? Showing 4 bars implies we can distinguish the signal level further than a binary signal/no signal distinction, which we can't. With such a channel we'd basically make a promise to the user, which we'd then fail to fulfill.

Reply:
Ok.

Reply:
Done